docs/index.rst

```diff
@@ -1018,7 +1017,9 @@ Example:
 ``model.json(...)``
 ~~~~~~~~~~~~~~~~~~~
 
-The ``json()`` method will serialise a model to JSON, ``json()`` in turn calls ``dict()`` and serialises its result.
+The ``json()`` method will serialise a model to JSON. Typically, ``json()`` in turn calls ``dict()`` and
```

Comment:
```suggestion
The ``.json()`` method will serialise a model to JSON. Typically, ``.json()`` in turn calls ``.dict()`` and
```

I think we should use `.dict()` in this whole section to distinguish from `dict(model)`, and therefore `.json()` to be consistent.